articles/service-fabric/how-to-managed-cluster-availability-zones.md

```diff
@@ -218,6 +218,32 @@ Requirements:
    ```
    If you run in to any problems reach out to support for assistance.
 
+## Enable FastZonalUpdate on Service Fabric managed clusters (preview)
+Service Fabric managed clusters support faster cluster and application upgrades by reducing the max upgrade domains per availability zone.   
+
+The update should be done via ARM template by setting the zonalUpdateMode property to “fast” snd then modify a node type attribute such as adding a node and then removing the node.  The Service Fabric managed cluster resource apiVersion should be 2022-10-01-preview or later.
```

Comment:
Can we use the same API version in the description and the sample template ?

Reply:
ack.  changed.